db/sql/src/cache.rs

```diff
@@ -60,7 +61,9 @@ pub struct HoprDbCaches {
     pub(crate) single_values: Cache<CachedValueDiscriminants, CachedValue>,
     pub(crate) unacked_tickets: Cache<HalfKeyChallenge, PendingAcknowledgement>,
     pub(crate) ticket_index: Cache<Hash, Arc<AtomicU64>>,
-    pub(crate) unrealized_value: Cache<Hash, Balance>,
+    // key is (channel_id, channel_epoch) to ensure calculation of unrealized value does not
+    // include tickets from other epochs
+    pub(crate) unrealized_value: Cache<(Hash, U256), Balance>,
```

Comment:
Why do we still need to keep the epoch as the key? 
We need to store only the unrealized value for the latest epoch and properly invalidate that on all places in the code, where epoch changes.

Reply:
We should not... just invalidate the cache... that will force the reload from the database, right? And because the tickets of the previous epoch are discarded,... you will get the correct new value.

Reply:
I'd much rather makes this as explicit as possible. The cleanup is done already as you mention.